app/controllers/stories_bulk_destroy_controller.rb

```diff
@@ -0,0 +1,24 @@
+class StoriesBulkDestroyController < ApplicationController
+  def create
+    authorize stories
+    stories.map do |story|
+      StoryOperations::Destroy.call(story, current_user)
+    end
+
+    redirect_to project, notice: "Stories was successfully destroyed."
```

Comment:
I wonder if we need a redirect here, this controller is probably going to be used in a ajax request, maybe HTTP 200 or HTTP 204?
Also we need to respond correctly when this operation fail right, so redirect wont be acceptable here